config/experiments.go

```diff
@@ -4,7 +4,6 @@ type Experiments struct {
 	FilestoreEnabled              bool
 	UrlstoreEnabled               bool
 	ShardingEnabled               bool `json:",omitempty"` // deprecated by autosharding: https://github.com/ipfs/kubo/pull/8527
-	GraphsyncEnabled              bool
```

Comment:
I could do like I usually do an make a special erroring type but I don't think it's very requiered, the error on the other wide when it will stop working should be pretty clear.